### PR TITLE
feat: $cCOP relayers deployment scripts

### PR DESCRIPTION
**Pull request by nvtaveras**
### Description

This adds the deployment scripts for deploying CELO/COP and COP/USD relayers, as well as the governance proposal script for whitelisting them in sorted oracles. Note that chainlink doesn't report COP/USD on Alfajores, so a mock aggregator was reported instead [at this address](https://alfajores.celoscan.io/address/0x8c07f12022FD7AfC700A410a0f109dD349d89B49#code). These are the deployed relayer addresses per network:

**Alfajores**
CELO/COP: https://alfajores.celoscan.io/address/0xab461A2742291515e126A353c05BA0587117060A
COP/USD: https://alfajores.celoscan.io/address/0x46D7Bb2cE340f677FD059181C77bAB3A3CDdAC09

**Mainnet**
CELO/COP: https://celoscan.io/address/0x5926F76D43Ce2D778880226b3C4e7156C8Ece99e
COP/USD: https://celoscan.io/address/0x783F947126Adb7646c2A459B867f5B526D2E6603

**Rate feed ids**
`relayed:CELOCOP`: `0x32ABF1cBdFdcD56790f427694be2658d4B1A83bC`
`relayed:COPUSD`: `0x0196D1F4FdA21fA442e53EaF18Bf31282F6139F1`

**Important**: I wanted to wait for reviews before deploying the relayers to mainnet but accidentally ran the deployment script, so they are already there. So **please double check that both relayers are wired correctly while reviewing this PR**. You can do this by going to Celoscan and regading the contract configuration: (_aggregators, maxTimestampSpread, description, rateFeedId_).

### Tested

- Checked manually that the configured relayers look good.
- Executed the COPWhitelist proposal on Alfajores
  > celocli oracle:list 0x32ABF1cBdFdcD56790f427694be2658d4B1A83bC
    [ '0xab461A2742291515e126A353c05BA0587117060A' ]
  > celocli oracle:list 0x0196D1F4FdA21fA442e53EaF18Bf31282F6139F1
    [ '0x46D7Bb2cE340f677FD059181C77bAB3A3CDdAC09' ]
- COPWhitelist checks simulation is green on Mainnet
  > yarn cgp -n celo -u COPWhitelist -g celo -s
  Script ran successfully.
  == Logs ==
  Deployed COPWhitelist to 0xb0b8c7cCc64be9aa9B5712BBEc5a5BD0a3213059
  Proposal was simulated successfully.
  Deployed COPWhitelistChecks to 0xB5A92EB54CD44C87875Ec8d4e166708ec6CCa61F
  ✅ CELO/COP and COP/USD relayers whitelisted correctly
  CELO/COP (CELO/USD:USD/COP): 3215.791668419945297706000000
  COP/USD: 0.000237650000000000000000
  ✅ CELO/COP and COP/USD relayers relayed successfully
  ✅ Token report expiry for rateFeedId [0x32ABF1cBdFdcD56790f427694be2658d4B1A83bC] is correct
  ✅ Token report expiry for rateFeedId [0x0196D1F4FdA21fA442e53EaF18Bf31282F6139F1] is correct

### Related issues

- Fixes https://github.com/mento-protocol/mento-general/issues/556

### CODE REVIEW

File: script/upgrades/COPWhitelist/deploy/COPWhitelist-00-Deploy-ChainlinkRelayers.sol [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    string rateFeedDescription;
+    uint256 maxTimestampSpread;
+    IChainlinkRelayer.ChainlinkAggregator[] aggregators;
+  }
+
+  Relayer[] private relayers = [
+    Relayer({
+      rateFeed: "relayed:CELOCOP",
+      rateFeedDescription: "CELO/COP (CELO/USD:USD/COP)",
+      maxTimestampSpread: 24 hours,
```
Comment (chapati23) [inline script/upgrades/COPWhitelist/deploy/COPWhitelist-00-Deploy-ChainlinkRelayers.sol:31]: Chainlink updated CELO/USD to also report every 5 minutes, so I think we could lower this now to 5 or 6 minutes

**Review by chapati23 — COMMENTED** [review]
see comment, CELO/USD heartbeat is now also 5mins so we could tighten up the `maxTimestampSpread` for CELO/XXX relayers now if we wanted to

File: script/upgrades/COPWhitelist/deploy/COPWhitelist-00-Deploy-ChainlinkRelayers.sol (hunk 1 above)
Comment (chapati23) [reply script/upgrades/COPWhitelist/deploy/COPWhitelist-00-Deploy-ChainlinkRelayers.sol:31]: Here's Chainlink's CELO/USD aggregator reporting every ~5mins now https://celoscan.io/address/0x5FB8bFAEBe2dd6dB2AF308D293ea25e607D3A922

Comment (bayological) [reply script/upgrades/COPWhitelist/deploy/COPWhitelist-00-Deploy-ChainlinkRelayers.sol:31]: Just going to keep this as is for now.

**Review by bayological — APPROVED** [review]

**Review by chapati23 — APPROVED** [review]
let's go

**Pull request merged**